File: noi.openjudge.cn/ch0104/18.cpp
// http://noi.openjudge.cn/ch0104/solution/39179017/

#include<iostream>
using namespace std;
int main() {
    int x,y;
    cin >> x >> y;
    if (x >= -1 && x <= 1) {
        if (y >= -1 && y <= 1) {
            cout << "yes" << endl;
        } else {
            cout << "no" << endl;
        }
    } else {
        cout << "no" << endl;
    }
}

File: noi.openjudge.cn/ch0105/18.py
# http://noi.openjudge.cn/ch0105/solution/39396563/

n = int(input())
x,y = map(int, input().split())
a = y / x * 100
for i in range(n-1):
    x,y = map(int, input().split())
    b = y / x * 100
    if b - a > 5:
        print("better")
    elif a - b < 5:
        print("same")
    else :
        print("worse")
